### DISCUSSION TITLE
Local notifications

### DISCUSSION THREAD

**Discussion by nk-alex** [shinyorg/shiny#1481] [Q&A]
I would like to integrate local notifications in my MAUI application. But I'm not able to find any sample on how to do it. On [docs](https://shinylib.net/client/notifications/) page it's not described.

**Comment by aritchie** [comment]
There are samples in the shiny repo: https://github.com/shinyorg/shiny/tree/dev/samples/Sample.Maui/Notifications.  Please note that if you aren't happy with the docs, you are welcome to research the source code as this is not a product.
